src/main/index.js

```diff
@@ -72,6 +74,7 @@ async function run(config) {
         '--volume', '/sys/kernel/debug:/sys/kernel/debug:ro',
         '--volume', '/home/runner/work:/github_workspace',
         '--env', `CIMON_LOG_LEVEL=${config.cimon.logLevel}`,
+        '--env', `CIMON_UPLOAD_ARTIFACT=${config.cimon.uploadArtifact}`,
```

Comment:
This flag isn't supported yet in Cimon. We can add it later on (on another PR) once we add it to Cimon

Reply:
Do you think we should remove it for now?